File: latihantujuh/src/main/java/com/android/cnd/latihantujuh/BiodataLinearActivity.java
package com.android.cnd.latihantujuh;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class BiodataLinearActivity extends AppCompatActivity {
    String title = "";
    TextView titleName;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_biodata_linear);
        titleName = findViewById(R.id.title);
        title = getIntent().getStringExtra("KEY_TITLE");
        titleName.setText(title);

    }
}

File: kotlin/latihantujuh/src/main/java/com/android/cnd/latihantujuh/BiodataLinearActivity.kt
package com.android.cnd.latihantujuh

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class BiodataLinearActivity : AppCompatActivity() {
    private lateinit var titleName: TextView
    private var title: String? = ""

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_biodata_linear)
        titleName = findViewById(R.id.title)
        title = intent.getStringExtra("KEY_TITLE")
        titleName.text = title
    }
}